e2e/test/scenarios/onboarding/setup/setup.cy.spec.js

```diff
@@ -304,6 +309,70 @@ describe("scenarios > setup", () => {
     cy.visit("/browse");
     cy.findByTestId("database-browser").findByText(dbName);
   });
+
+  it("embedded use-case, it should hide the db step and show the embedding homepage", () => {
+    cy.visit("/setup");
+
+    cy.location("pathname").should("eq", "/setup");
```

Comment:
this is a copy and paste of another test, I will refactor them later to extract common bits, for now I wanted to get this merged/have feedback earlier